packages/block-editor/src/hooks/duotone.js

```diff
@@ -140,12 +140,33 @@ ${ selector } {
 	);
 }
 
+function useMultiOriginPresets( { presetSetting, defaultSetting } ) {
+	const disableDefault = ! useSetting( defaultSetting );
+	const userPresets =
+		useSetting( `${ presetSetting }.custom` ) || EMPTY_ARRAY;
+	const themePresets =
+		useSetting( `${ presetSetting }.theme` ) || EMPTY_ARRAY;
+	const defaultPresets = disableDefault
+		? EMPTY_ARRAY
+		: useSetting( `${ presetSetting }.default` ) || EMPTY_ARRAY;
```

Comment:
We should avoid conditionally calling hooks. It's a bit awkward but we can re-arrange this to something like:

```jsx
const defaultPresetsSetting =
useSetting( `${ presetSetting }.default` ) || EMPTY_ARRAY;
const defaultPresets = disableDefault ? EMPTY_ARRAY : defaultPresetsSetting;
```